kubejs/client_scripts/jei_hide.js

```diff
@@ -49,6 +48,37 @@ JEIEvents.hideItems(event => {
 	]
 	backpack.forEach(element => event.hide(element))
 
+	const storages = [
+		'sophisticatedstorage:advanced_feeding_upgrade',
+		'sophisticatedstorage:advanced_magnet_upgrade',

```

Comment:
Instead of removing the advanced feeding upgrade and the advanced magnet upgrade, could you make these more difficult to obtain than the non-advanced upgrades? Maybe gate them both to HV?